Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/Login/LoginWall.swift

```diff
@@ -21,8 +21,10 @@ struct LoginWall<ContentView: View>: View {
         case .unknown:
             ProgressView()
                 .displayError(self.$error)
-                .task { @MainActor in
-                    await reload()
+                .onAppear {
+                    Task {
+                        await reload()
+                    }
```

Comment:
For some reason the .task caused an error (and subsequent error dialog) to pop up when run on iOS 15 (worked as expected on 16 and 17).

Reply:
Can you leave a comment in there for this? 😅  This is something that one of us (probably me) would change in the future and not realize why we did this 😛

Reply:
Good idea, will do!

Reply:
Done!